Makefile

```diff
@@ -2,6 +2,7 @@ BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 VERSION=1.0.0-alpha.1
+TODAY=$(shell date +"%d-%m-%Y-%H-%M-%S")
```

Comment:
@hferentschik you want sha+date+ci_build_id as part of BUILD_ID? provided ci_build_id present as env and for  jenkins it will be there. otherwise use 'local'.